util/util.go

```diff
@@ -70,3 +71,19 @@ func Exists(name string) bool {
 	}
 	return true
 }
+
+// CommandRunner will execute the commands and return the results of that
+// execution.
+type CommandRunner interface {
```

Comment:
This shouldn't be in util, it's not common. just keep in in packages. For the other libraries that need something similar we should create their own interface. The contains methods that are only ever used in one package, i don't want other packages to have to implement those.

Reply:
yeah.. i just realized, runwithpty would be very specific to package package.